tasks/config/run.js

```diff
@@ -40,107 +40,82 @@ const OPENSEARCH_DASHBOARDS_INSTALL_DIR =
 module.exports = function () {
   const NODE = 'node';
   const YARN = 'yarn';
-  const scriptWithGithubChecks = ({ title, options, cmd, args }) =>
-    process.env.CHECKS_REPORTER_ACTIVE === 'true'
-      ? {
-          options,
-          cmd: YARN,
-          args: ['run', 'github-checks-reporter', title, cmd, ...args],
-        }
-      : { options, cmd, args };
-  const gruntTaskWithGithubChecks = (title, task) =>
-    scriptWithGithubChecks({
-      title,
-      cmd: YARN,
-      args: ['run', 'grunt', task],
-    });
 
   return {
     // used by the test and jenkins:unit tasks
     //    runs the eslint script to check for linting errors
-    eslint: scriptWithGithubChecks({
-      title: 'eslint',
+    eslint: {
       cmd: NODE,
       args: ['scripts/eslint', '--no-cache'],
-    }),
+    },
 
-    sasslint: scriptWithGithubChecks({
-      title: 'sasslint',
+    stylelint: {
       cmd: NODE,
-      args: ['scripts/sasslint'],
-    }),
+      args: ['scripts/stylelint'],
+    },
```

Comment:
This should remain as `sasslint`; simply replace `stylelint` with `sasslint`:
```suggestion
    sasslint: {
      cmd: NODE,
      args: ['scripts/sasslint'],
    },
```

Reply:
thx, made the update